Index: Makefile
===================================================================
--- Makefile.orig	2010-09-05 16:32:10.000000000 +0200
+++ Makefile	2011-03-06 16:17:54.595150499 +0100
@@ -35,8 +35,8 @@ export CXX ?= g++
 export AS := nasm
 export RANLIB ?= ranlib
 
-export CFLAGS := -Wall
-export CXXFLAGS := -Wall -Wno-unused-parameter
+export CFLAGS := $(CFLAGS) -Wall
+export CXXFLAGS := $(CXXFLAGS) -Wall -Wno-unused-parameter
 C_CXX_FLAGS := -MMD -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGE_FILES -I$(BASE_DIR) -I$(BASE_DIR)/Crypto
 export ASFLAGS := -Ox -D __GNUC__
 export LFLAGS :=
